shopfloor_mobile/static/wms/src/scenario/single_pack_transfer.js

```diff
@@ -75,18 +75,23 @@ const SinglePackTransfer = {
                     :record="state.data"
                     :card_color="utils.colors.color_for('screen_step_done')"
                     />
-                <item-detail-card
-                    :key="make_state_component_key(['product', state.data.id])"
-                    :record="state.data"
-                    :options="utils.wms.move_line_product_detail_options()"
-                    :card_color="utils.colors.color_for('screen_step_done')"
-                    />
+                <v-card
+                    :color="utils.colors.color_for('screen_step_done')"
+                    class="mb-2"
+                >
+                    <v-card-text
+                        v-for="product in state.data.products"
+                        class="pt-2 pb-2"
+                    >
+                        <b>{{ product.name}}</b>
+                    </v-card-text>
+                </v-card>
```

Comment:
I don't like the separate card for the products. It's a detail of the package and should be shown inside it:

```
                <item-detail-card
                    :key="make_state_component_key(['package', state.data.id])"
                    :record="state.data"
                    :card_color="utils.colors.color_for('screen_step_done')"
                    >
                    <template v-slot:after_details>
                        <v-card-text class="details pt-0">
                            <div v-for="product in state.data.products" class="field-detail">
                                {{ product.display_name}}
                            </div>
                        </v-card-text>
                    </template>
                </item-detail-card>
```